how-tos/users-in-one-org.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: "Setting a Maximum Number of Users"
```

Comment:
Is there another way to phrase this so it's obvious it means "...within an organization"